openmc/cell.py

```diff
@@ -563,7 +563,7 @@ def create_surface_elements(node, element):
                 if isinstance(node, Halfspace):
                     path = "./surface[@id='{}']".format(node.surface.id)
                     if xml_element.find(path) is None:
-                        xml_element.append(node.surface.create_xml_subelement())
+                        xml_element.append(node.surface.to_xml_element())
```

Comment:
I like the shorter `to_xml_*()` name better. Perhaps everything should be converted to this naming convention in the future.